cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -90,6 +90,7 @@ private class ServiceImpl(
                     CCPA -> {
                         dataStorage.saveCcpaConsentResp(it.userConsent ?: "")
                         dataStorage.saveCcpaConsentUuid(it.uuid)
+                        dataStorage.saveUsPrivacyString(it.content.getJSONObject("userConsent").getString("uspstring")?:"")
```

Comment:
If the `userConsent` is not there, the client application will crash directly, please add some check before using the `getJSONObject` api

Reply:
@carmelo-iriti done, check it out!

Reply:
We are getting there 👍🏻
1. Let’s move the extensions in the `ConsentRespExt.kt` because they are part of the consent not of the Service
2. Inside the `ConsentRespExtKtTest.kt` we need 2 tests, 
      a. one will use a JSONObject with the following content and the Ext check that it gets `1---` 
`{
  "consents": {
    "USPString": "1---"
  }
}`
   b. one will use an empty JSONObject() and the Ext check that it gets `null`